src/beautifiers/index.coffee

```diff
@@ -70,6 +70,7 @@ module.exports = class Beautifiers extends EventEmitter
     'yapf'
     'erl_tidy'
     'marko-beautifier'
+    'r-beautifier'
```

Comment:
As I explain below I had lots of trouble with `npm run docs` because of this. I will try changing both index.coffee to formatR and see what happens rather than change them to R beautifier, as it is now.